Monika After Story/game/script-topics.rpy

```diff
@@ -7591,6 +7591,127 @@ label monika_cartravel:
     m 1hua "Just the idea of travelling around the world makes me so excited!"
     m 1eka "I really can't wait, [player]~"
     return
+    
+    ###poems archives
+    
+    default persistent._mas_pm_ask_poems = None #first time player asks
+    
+    
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="show_poem",category=['Ddlc'],prompt="Can you show me poem?",random=False))
+    
+label show_poem:
+    m 1eud "You want to see a poem from the game?"
+    if persistent._mas_pm_ask_poems:
+        m 1hua "All right, let me search the file..."
```

Comment:
I feel like this is a bit immersion breaking when she mentions searching the file. I think it would be better if she just said she had to look for the poem itself. ie. The piece of paper.

Reply:
Ok, I changed it

> m 1hua "All right, let me search..."